orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf

```diff
@@ -20,6 +20,7 @@ module "orc8r" {
   orc8r_db_password           = "mypassword" # must be at least 8 characters
   secretsmanager_orc8r_secret = "orc8r-secrets"
   orc8r_domain_name           = "orc8r.example.com"
+  orc8r_sns_email             = "admin@example.com"
```

Comment:
minor suggestion - add `enable_aws_db_notifications` also to this example.

Reply:
Added enable_aws_db_notifications to main.tf